src/app/container/container.component.html

```diff
@@ -185,7 +185,15 @@ <h3>
         </mat-tab>
       </mat-tab-group>
     </div>
-    <div *ngIf="publicPage" class="mt-2" id='discourse-comments'></div>
+
+    <div *ngIf="publicPage" class="mt-2 mr-3">
+      <div *ngIf="tool?.topicId !== null else noTopicId" id='discourse-comments'></div>
+      <ng-template #noTopicId>
+        <mat-card class="alert alert-warning" role="alert">
+          <mat-icon>info</mat-icon> No Discourse topic exists for this tool. Contact a Dockstore curator to create a topic.
```

Comment:
How do you contact a Dockstore curator? I don't even know myself. :) Could this be a link?

Reply:
That is a good question :laughing: